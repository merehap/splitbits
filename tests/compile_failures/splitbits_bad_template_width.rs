use splitbits::splitbits;

fn main() {
    let input = 5;
    splitbits!(input, "aaaabbb");
}
