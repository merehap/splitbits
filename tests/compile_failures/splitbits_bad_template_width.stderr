error: proc macro panicked
 --> tests/compile_failures/splitbits_bad_template_width.rs:5:5
  |
5 |     splitbits!(input, "aaaabbb");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Template must have a valid width: "Template width must be 8, 16, 32, 64, or 128, but was 7."
